auth-web/src/store/modules/business.ts

```diff
@@ -104,6 +104,9 @@ export default class BusinessModule extends VuexModule {
         header: {
           name: FilingTypes.INCORPORATION_APPLICATION,
           accountId: accountId
+        },
+        business: {
+          legalType: LegalTypes.BCOMP
```

Comment:
Im guessing, future state this prop is applied dynamically but BCOMPs for MVP?

Reply:
Yep, exactly.